promql/parse_test.go

```diff
@@ -952,6 +965,22 @@ var testExpr = []struct {
 		input:  `foo{__name__="bar"}`,
 		fail:   true,
 		errMsg: "metric name must not be set twice: \"foo\" or \"bar\"",
+	}, {
+		input:  `foo{__name__= =}`,
+		fail:   true,
+		errMsg: "unexpected <op:=> in label matching, expected string",
+	}, {
+		input:  `foo{,}`,
+		fail:   true,
+		errMsg: "unexpected \",\" in label matching, expected identifier or \"}\"",
```

Comment:
You can use ` to avoid all the escaping.

Reply:
True.

I've changed the tests accordingly.

However I'd like to leave it as it is in the yacc grammar, since using ` messes up syntax highlighting in yacc files (tested with Vim and VS Code).